chrome_manifest_v3/class_model.js

```
@@ -1,4 +1,4 @@
-var G2T = G2T || {}; // must be var to guarantee correct scope
+var G2T = G2T || {}; // must be var to guarantee correct scope - do not alter this line
```

Biome (2.1.2)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)